lib/perl/Genome/Model/ClinSeq/Util.pm

```diff
@@ -1264,27 +1264,30 @@ sub get_ref_align_builds{
   }
 
   #Determine the time point position
-  my %timepoint_positions;
+  my %time_point_positions;
   foreach my $name (sort {$ref_builds{$a}->{time_point} cmp $ref_builds{$b}->{time_point}} keys %ref_builds){
-    my $day = $ref_builds{$name}{day};
-    if ($day =~ /day(\d+)/){
-      my $day_number = $1;
-      $timepoint_positions{$day_number}{position} = 0;
-      $ref_builds{$name}{day_number} = $day_number;
-    }else{
-      die $self->error_message("could not parse day value from sample attribute (caTissue timepoint): $day");
+    my $time_point_string = $ref_builds{$name}{time_point_string};
+    if ($time_point_string =~ /(day|Surgery)(\d+)/){
```

Comment:
I am ok with the suggestion; however, it could result in unexpected values in downstream plots.  Regardless, I don't think this is the right place to attempt to control the time point vocabulary.  I think it's ok to use the values found in the database as-is.  @malachig any thoughts on how open you want ClinSeq to be to new caTissue time point values?